src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -544,6 +546,15 @@ private void WriteProtocolMethodDocumentationWithExternalXmlDoc(LowLevelClientMe
             }
         }
 
+        private void WriteConvenienceMethodOmitReasonIfNecessary(ConvenienceMethodOmittingMessage? message)
+        {
+            // TODO -- create wiki links to provide guidance here: https://github.com/Azure/autorest.csharp/issues/3624
+            if (message == null)
+                return;
+
+            _writer.Line($"// {message.Message}");
```

Comment:
I am pretty sure this is the case, but can we verify that this won't show up in docs / intellisense?  If it does move it to the first line of the method.

Reply:
It does not show up in intellisense:
![image](https://github.com/Azure/autorest.csharp/assets/10554446/c30d3395-468b-4ab0-ac51-e752959ca599)
Therefore I assume it will not show up in docs neither